### PR TITLE
146 fix swagger documentation

### LINKED ISSUE #146: Fix OpenAPI docs
The [endpoint documentation](http://localhost:8080/api/documentation/index.html) has no content right now.

### PR DESCRIPTION
**Pull request by mathiashoeld**
Closes #146

### CODE REVIEW

File: api/src/group_create.ts [hunk 1]
```diff
@@ -46,63 +46,62 @@ function validateRequestBody(body: any): Result.Type<RequestBody> {
...
-              code: { type: "string", example: "409" },
-              message: { type: "string", example: "User already exists." },
+      response: {
+        200: {
+          description: "successful response",
+          type: "object",
+          properties: {
+            apiVersion: { type: "string", example: "1.0" },
+            data: {
+              type: "object",
```
Comment (Stezido) [inline api/src/group_create.ts]: We should return information about the created group on success

File: api/src/notification_mark_read.ts [hunk 2]
```diff
@@ -37,34 +37,43 @@ function validateRequestBody(body: any): Result.Type<RequestBody> {
...
-    response: {
-      204: { description: "successful response" },
-      401: NotAuthenticated.schema,
+      response: {
+        200: {
+          description: "successful response",
+          type: "object",
+          properties: {
+            apiVersion: { type: "string", example: "1.0" },
+            data: {},
```
Comment (Stezido) [inline api/src/notification_mark_read.ts]: type: "object" is missing here

File: api/src/system/restoreBackup.ts [hunk 3]
```diff
@@ -34,7 +34,7 @@ export const restoreBackup = async (
     200,
     {
       apiVersion: "1.0",
-      data: "OK",
+      data: {},
```
Comment (Stezido) [inline api/src/system/restoreBackup.ts]: change it in schema too

File: api/src/network/controller/registerNode.ts [hunk 4]
```diff
@@ -37,6 +37,6 @@ export async function registerNode(multichain: MultichainClient, req): Promise<H
   } else {
     await Nodes.publish(multichain, address, event);
     logger.info(`Node ${address} registered`);
-    return [200, { apiVersion: "1.0", data: "OK" }];
+    return [200, { apiVersion: "1.0", data: {} }];
```
Comment (Stezido) [inline api/src/network/controller/registerNode.ts]: change it in schema too

File: api/src/network/controller/vote.ts [hunk 5]
```diff
@@ -47,7 +47,7 @@ export async function voteForNetworkPermission(
     logger.debug("No action required.");
   }
 
-  return [200, { apiVersion: "1.0", data: "OK" }];
+  return [200, { apiVersion: "1.0", data: {} }];
```
Comment (Stezido) [inline api/src/network/controller/vote.ts]: change it in schema too

Comment (Stezido) [reply api/src/network/controller/vote.ts]: same story for network.approveNewOrganization and network.approveNewNodeForExistingOrganization

File: api/src/notification_mark_read.ts (hunk 2 above)
Comment (mathiashoeld) [reply api/src/notification_mark_read.ts]: changed to: 
```
data: { type: "object" }
```

File: api/src/system/restoreBackup.ts (hunk 3 above)
Comment (mathiashoeld) [reply api/src/system/restoreBackup.ts]: changed in `schema.ts`: 
```
data: { type: "object" }
```

File: api/src/network/controller/registerNode.ts (hunk 4 above)
Comment (mathiashoeld) [reply api/src/network/controller/registerNode.ts]: changed in `schema.ts:2988`: 
```
data: {type: "object"}
```

File: api/src/network/controller/vote.ts (hunk 5 above)
Comment (mathiashoeld) [reply api/src/network/controller/vote.ts]: changed data type to `object` in `schema.ts` for the following: 

- voteForPermission (line 3032)
- approveNewOrganization (line 3071) 
- approveNewNodeForExistingOrganization (line 31112)

File: api/src/group_create.ts (hunk 1 above)
Comment (mathiashoeld) [reply api/src/group_create.ts]: what kind of information should be returned? 
for e.g. projects we return the project id - in the case of groups, the group id is part of the request. should it then still be returned?

Comment (mathiashoeld) [reply api/src/group_create.ts]: changed returned information to following object:
```
group: {
                  type: "object",
                  properties: {
                    id: { type: "string", example: "d0e8c69eg298c87e3899119e025eff1f" },
                    displayName: { type: "string", example: "mstein" },
                    users: { type: "array", items: { type: "string" } },
                  },
                },
```

**Pull request merged**